plugins/doc_fragments/redis.py

```diff
@@ -45,7 +45,7 @@ class ModuleDocFragment(object):
     default: true
   ca_certs:
     description:
-      - Path to root certificates file. If not set and O(tls) is set to V(true), certifi ca-certificates will be used.
+      - Path to root certificates file. If not set and O(tls) is set to V(true), certifi's C(ca-certificates) is used.
```

Comment:
There does not exist anything in certifi that's called `ca-certificates`. I think this either should be two words "CA certificates", or denote the file `cacert.pem` included with certifi. I think the former is better:
```suggestion
      - Path to root certificates file. If not set and O(tls) is set to V(true), certifi's CA certificates are used.
```